src/pybind/mgr/dashboard/controllers/osd.py

```diff
@@ -44,7 +44,8 @@ def osd_task(name, metadata, wait_for=2.0):
 @ApiController('/osd', Scope.OSD)
 @ControllerDoc('OSD management API', 'OSD')
 class Osd(RESTController):
-    def list(self):
+    @handle_send_command_error('osd')
```

Comment:
Is this decorator still needed?

Reply:
It was as per the previous implementation, deleted it now, Thanks :)